extra-cpp-bindings/src/lib.rs

```diff
@@ -75,24 +75,27 @@ mod ffi {
 
     /// The wallet registry entry
     pub struct WalletEntry {
+        /// wallet id
+        pub id: String,
         /// its name
         pub name: String,
         /// icon URLs
         pub image_url: ImageUrl,
-        /// native link (Android/Desktop), empty if none
-        pub native_link: String,
-        /// universal link (iOS), empty if none
-        pub universal_link: String,
+        /// mobile native link, empty if none
+        pub mobile_native_link: String,
+        /// mobile universal link, empty if none
+        pub mobile_universal_link: String,
+        /// desktop native link, empty if none
+        pub desktop_native_link: String,
+        /// desktop universal link, empty if none
+        pub desktop_universal_link: String,
```

Comment:
why is it duplicated? Are there wallets that have different links for desktop/mobile?

Reply:
Yes. They are not quite the same.

Reply:
Mainly because wallets some have empty or null on mobile, some have empty or null on desktop, they have no relationships.